lib/publiccloud/instance.pm

```diff
@@ -46,8 +47,10 @@ sub run_ssh_command {
 
     my $ssh_cmd = sprintf('ssh %s -i "%s" "%s@%s" -- %s',
         $self->ssh_opts, $self->ssh_key, $self->username, $self->public_ip, $args{cmd});
-    record_info('CMD', $ssh_cmd);
-    return script_output($ssh_cmd, $args{timeout}, %args);
+    record_info('SSH CMD', $ssh_cmd) unless ($args{quiet});
```

Comment:
I would still do the record_info even for quiet=>1 . it's not like serial output which might be too much sometimes, record_info is always good to have.

Reply:
ok, sounds reasonable, just to be on the safe side.